release/scripts/UpdateGwasStudies.py

```diff
@@ -37,6 +35,26 @@ def get_gwas_info(self,sample):
         if response_data:
             try:
                 source_PMID = response_data['publicationInfo']['pubmedId']
+
+                # Create list of cohorts if it exists in the GWAS study
+                # This override the Cohorts found previously in the cohort column in the spreadsheet
```

Comment:
Can we get a warning/message if the spreadsheet cohorts are replaced? (including previous and new values)

Reply:
It will look like that:
```
# GCST00XXXX:
	1 distinct ancestries
	{'source_PMID': '1234567', 'sample_number': 23464, 'ancestry_broad': 'European', 'ancestry_country': 'U.K.'}
	/!\ Replacing cohorts list:
	  - Old set: ACTS, LASA
	  + New set: UKB
	>> SCORE updated: PGS00XXXX
```
with the new bits:
```
/!\ Replacing cohorts list:
  - Old set: ACTS, LASA
  + New set: UKB
```

Reply:
It might useful to know if it's just additions or things are missing? It's more likley to be correct if it's adding more annotations, but if it's removing an annotation we should be careful.

Reply:
If it's using the GCST, it should match what is in the GWAS Catalog ? 
At the moment this script overwrite the Sample if the `sample_number`, `sample_cases` and `sample_controls` are **NULL**, i.e. GCSTs which were not released when the study was imported.